userprofile/models.py

```diff
@@ -171,6 +181,15 @@ class HealthCheckUserProfile(
     rooms_in_household = models.IntegerField(blank=True, null=True, default=None)
     persons_in_household = models.IntegerField(blank=True, null=True, default=None)
     language = models.CharField(max_length=3, null=True, blank=True)
+    hcs_study_a_arm = models.CharField(
+        max_length=3, choices=StudyArm.choices, null=True
+    )
+    hcs_study_b_arm = models.CharField(
+        max_length=3, choices=StudyArm.choices, null=True
+    )
+    hcs_study_c_arm = models.CharField(
+        max_length=3, choices=StudyArm.choices, null=True
+    )
```

Comment:
Do we need a `default=None` here too? Not sure what the value of these fields will be after migration, and we want them to be null for a quick migration time.